README.md

```diff
@@ -175,67 +164,69 @@ Just like Django Crispy Forms, you need to pass your form object to the
 
 ```html
 {% load crispy_forms_tags %}
+
 <html>
     <body>
-        {% crispy your_form %}
+        {% crispy your_sexy_form %}
     </body>
 </html>
 ```
 
-### Add a submit button and customise the form via the `helper` property
+### `FormHelper`s
 
-Submit buttons are not automatically added - you will need to do this by
-extending the form helper's `layout` (example below).
+A [FormHelper](https://django-crispy-forms.readthedocs.io/en/latest/form_helper.html)
+allows you to alter the rendering behaviour of forms.
 
-Every form that inherits from `TbxFormsBaseForm` will have the following
-attributes set:
+Every form that inherits from `TbxFormsBaseForm` (i.e. every form within `tbxforms`)
+will have a `FormHelper` with the following default attributes:
 
 -   `html5_required = True`
 -   `label_size = Size.MEDIUM`
 -   `legend_size = Size.MEDIUM`
 -   `form_error_title = _("There is a problem with your submission")`
 -   Plus everything from [django-crispy-forms' default attributes](https://django-crispy-forms.readthedocs.io/en/latest/form_helper.html).
 
-These can be overridden (and/or additional attributes from the above list defined)
-just like you would do with any other inherited class, e.g.:
+These can be changed during instantiation or [on the go](https://django-crispy-forms.readthedocs.io/en/latest/dynamic_layouts.html) - examples below.
+
+#### Add a submit button
+
+Submit buttons are not automatically added to forms. To add one, you can extend
+the `form.helper.layout` (examples below).
+
+Extend during instantiation:
 
 ```python
 from django import forms
-from wagtail.contrib.forms.forms import BaseForm as WagtailBaseForm
 from tbxforms.forms import BaseForm as TbxFormsBaseForm
```

Comment:
This could be a separate issue but I've just realised that `tbxforms.forms.BaseForm` is actually a mixin: https://github.com/torchbox/tbxforms/blob/main/tbxforms/forms.py#L14 So maybe it should be called that instead of `BaseForm`? Also, to use only one superclass across the codebase one might create an actual `BaseForm` (inheriting from both tbxform and django forms) in a given project, so the naming will become even more confusing.